tests/benchmarks/multicore/idle/boards/nrf54h20dk_nrf54h20_cpuapp_ram_high_usage.overlay

```diff
@@ -20,3 +20,11 @@
 &usbhs {
     status = "disabled";
 };
+
```

Comment:
It does not help for high current consumption...

Reply:
I will revert this commit , the low level tests are passing now but the reason that they failed before was not that I enabled these dts entries.